Dockerfile

```diff
@@ -35,13 +38,8 @@ RUN /tmp/provision-web.sh
 WORKDIR /code
 ADD . /code
 
-# Run container as user `invenio` with UID `1000`, which should match
-# current host user in most situations:
-RUN adduser --uid 1000 invenio && \
-    chown -R invenio:invenio /code
-USER invenio
-
 # Configure CERN Open Data Portal instance:
+ENV INVENIO_INSTANCE_PATH=/usr/local/var/cernopendata/var/cernopendata-instance
```

Comment:
Since we use `INVENIO_<host>_variable` convention for the various `INVENIO_*_*` environment variables for installation, what about using `INVENIO_WEB_INSTANCE_PATH`.